cohorts/load.py

```diff
@@ -568,26 +576,33 @@ def load_polyphen_annotations(self, as_dataframe=False):
         """
         patient_annotations = {}
         for patient in self:
-            annotations = self._load_single_patient_polyphen(patient)
-            annotations["patient_id"] = patient.id
-            patient_annotations[patient.id] = annotations
+            annotations = self._load_single_patient_polyphen(
+                patient,
+                filter_fn=filter_fn)
+            if annotations is not None:
+                annotations["patient_id"] = patient.id
+                patient_annotations[patient.id] = annotations
         if as_dataframe:
             return pd.concat(patient_annotations.values())
         return patient_annotations
 
-    def _load_single_patient_polyphen(self, patient):
+    def _load_single_patient_polyphen(self, patient, filter_fn):
         cache_name = self.cache_names["polyphen"]
         cached_file_name = "polyphen-annotations.csv"
+        variants = self._load_single_patient_variants(patient,
+                                                      variant_type="snv",
+                                                      merge_type="union",
+                                                      filter_fn=None)
```

Comment:
That gets handled in `filter_polyphen_with_metadata`; if we filtered variants here, we'd cache the filtered annotations vs. all of them. It's confusing.